keyboards/boardwalk/readme.md

```diff
@@ -0,0 +1,16 @@
+# Boardwalk
+
+![Boardwalk](https://i.imgur.com/CQj3b9E.jpg)
+
+The Boardwalk is a 60% ortholinear keyboard, designed around Ergodox keycap sets and to fit into many standard 60% cases. The project was inspired by OLKB’s Atomic keyboard, which used larger 2u mods, but u/shensmobile
+decided to switch to 1.5u keys so that Ergodox sets would provide excellent compatibility. The rest of the board can be covered using standard key sizes from 60% sets.
+
+Keyboard Maintainer: QMK Community  
+Hardware Supported: Boardwalk Ortholinear PCB  
+Hardware Availability: [boardwalk-group-buy](https://www.panc.co/boardwalk-group-buy.html)
```

Comment:
Custom here is to note the website or store URL as the link text.

```suggestion
Hardware Availability: [panc.co](https://www.panc.co/boardwalk-group-buy.html)
```

Reply:
Good spot, thanks.